lib/Translation_unit.mli

```diff
@@ -11,25 +11,17 @@
 
 type error
 
-val parse_and_format_impl :
-     ?output_file:string
+val parse_and_format :
+     'a list Migrate_ast.Mapper.fragment
```

Comment:
```suggestion
     _ list Migrate_ast.Mapper.fragment
```

To show that we're not using the result

Reply:
it's possible to increase the polymorphism to `_ fragment` by replacing the `List.is_empty` call somewhere in `Translation_unit` by a `is_empty : _ fragment -> bool` call

Reply:
I simplified the type but didn't find the is_empty call you are speaking about.

Reply:
come to think of it, `_ list` is good enough. if fragments are extended to something that's not a list, we'll need to change this area anyway. The call site I had in mind is:

https://github.com/ocaml-ppx/ocamlformat/blob/f0aa54b429f2a8d6b1b84686f6e2c6d6161fa89a/lib/Fmt_ast.ml#L4538

(but it's already exposing a `'a list fragment`)